js/src/components/shipping-rate-section/flat-shipping-rates-input-cards.js

```diff
@@ -2,22 +2,78 @@
  * Internal dependencies
  */
 import isNonFreeFlatShippingRate from '.~/utils/isNonFreeFlatShippingRate';
-import FreeShippingCards from './free-shipping-cards';
 import EstimatedShippingRatesCard from './estimated-shipping-rates-card';
+import OfferFreeShippingCard from './offer-free-shipping-card';
+import MinimumOrderCard from './minimum-order-card';
 
 const FlatShippingRatesInputCards = ( props ) => {
-	const { audienceCountries, value, onChange = () => {} } = props;
-	const displayFreeShippingCards = value.some( isNonFreeFlatShippingRate );
+	const { audienceCountries, formProps } = props;
+	const { getInputProps, values, setValue } = formProps;
+	const displayFreeShippingCards = values.shipping_country_rates.some(
+		isNonFreeFlatShippingRate
+	);
+
+	const handleShippingRatesChange = ( onChange ) => ( newShippingRates ) => {
```

Comment:
💅 
`handleShippingRatesChange` is not exactly a handler, would it be a bit better to use another naming like `createShippingRatesChangeHandler`?